### ISSUE TITLE
Server responses docs

### ISSUE THREAD

**Issue by ghost** [unknown321/mgsv_emulator#15]
Hi, unknown321!

I would like very hard to try and make a fake server for archiving purposes.

I'm excited that the main game works in rpcs3, and although the online experience (fobs, leagues and resources, not the online multiplayer game) is quite small compared to the story, I think it would be very nice to have a custom server that enables the player to login, develop resources and so on with minimal changes client-side.

So for example, one could configure a fake enemy fob in this server, and that fob configuration will be send back to the client when opening the fob menu.

Of course, this means most of the commands must be implemented, in particular: All of the login sequence, and more from the fob & resources game. Leagues don't make sense here because there will be no real players (unless this server is made public for people to play freely, but I don't think a many will be interested)

So, TL;DR: is there any site that have the information of the commands formats, apart from mgsv_emulator?

**Comment by unknown321** [comment]
As far as I know, there are no other sites with info you ask for. You can figure out response format yourself by making a decoding-encoding proxy.

See https://github.com/unknown321/mgsv_nuke_watcher/blob/master/app.py and https://github.com/unknown321/mgsv_emulator/blob/master/emulator/client_commands.py for some command examples.
